charts/nginz/values.yaml

```diff
@@ -162,6 +162,10 @@ nginx_conf:
       disable_zauth: true
       envs:
       - staging
+    - path: /api-federation/swagger-ui
```

Comment:
can you update https://docs.wire.com/understand/api-client-perspective/swagger.html to mention this?